### DISCUSSION TITLE
Passwort für: Edimax Smart Plug SP-2101W V3

### DISCUSSION THREAD

**Discussion by vladisoft1982** [camueller/SmartApplianceEnabler#358] [Allgemein]
Hallo,

der Zugang mit: username=admin, password=1234 oder 12345678 scheint bei der SP-2101W **V3** nicht zu funktionieren.

Jede Anrage über http an das "Device", kommt mit dem dem http-Fehler 401 zurück. 

Hat jemand eine Idee, welches Passort bei der Version 3 verwendet wird? Bzw. wie man das Passwort herausfindet? Edilife APP funktioniert bei mir im Android...

Gruß,

Vladi

**Comment by camueller** [comment]
Zur SP-2101W  hat schon länger niemand gefragt, aber wenn ich mich richtig erinnere, haben die irgendwann die HTTP-Authentication-Methode umgestellt. Weil der SAE nur HTTP Basic Authentication unterstützt, kann die SP-2101W zumindiest mit v3 nicht vom SAE bentuzt werden.

Vielleicht hat jemand bessere/genauere Informationen ...

**Comment by vladisoft1982** [comment]
Danke! Ja, scheinbar haben die seit der Version 3 alles umgestellt. Habe eine Anleitung gefunden, wie man mit telnet an das Passwort drankommt.
https://www.jochen-scheib.de/edimax-smart-plug-enable-telnet-access/
Den Telnet port 1355 haben sie aber anscheinend auch dicht gemacht. Habe mit nmap auch kein weiteren Port außer 10000 offen gefunden..

**Comment by vladisoft1982** [comment]
übrigens: das Passwort 1234 bleibt solange, bis man die Steckdose mit dem App "EdiSmart" verbindet. Problem dabei ist: das Gerät hat eine fixe IP-Adresse, die ich nicht ändern kann...
